app/Http/Resources/VictimResource.php

```diff
@@ -24,10 +24,19 @@ public function toArray(Request $request): array
             'photo_url' => $this->photo_url,
             'status' => $this->status,
             'holding_location' => $this->holding_location?->name,
+            'last_known_location' => $this->last_known_location,
             'security_organ' => $this->security_organ?->name,
             'time_taken' => $this->status_date?->format('H:i d-m-Y'),
             'time_taken_formatted' => $this->status_date?->toDayDateTimeString(),
             'notes' => $this->notes,
+            'remanded_from' => $this->remanded_from?->name,
```

Comment:
@Marshud See changes made to holding_location and security_organ relationships - which resolved the issue

Reply:
@ssmusoke these have been updated in an updated PR